_includes/feed.liquid

```diff
@@ -33,11 +33,22 @@ section: smartContract
     }
 </style>
 
+<div id="rollups-note">
```

Comment:
Can we use Markdown & metadata to render this in the liquid layer instead of client side? 
Then we can just specify in the metadata whether or not a page needs a rollup-note in the metadata / frontmatter.

Reply:
Sure